assets/helpers/tracking/googleTagManager.js

```diff
@@ -29,6 +29,8 @@ type GaEventData = {
   label: ?string,
 }
 
+const gaPropertyId = 'UA-51507017-5';
```

Comment:
Shouldn't this come from the app configuration?

Reply:
When pairing with @Ap0c, we decided to declare it in line, partly because as far a I'm aware, this value _isn't_ configurable. Would be keen to get Jamie's opinion on this.